# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtk-perl10/gtkhtml_fixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur Gtk-Perl-0.7008/GtkHTML/pkg.defs Gtk-Perl-0.7008.test/GtkHTML/pkg.defs
--- Gtk-Perl-0.7008/GtkHTML/pkg.defs	2001-01-21 12:21:06.000000000 +0100
+++ Gtk-Perl-0.7008.test/GtkHTML/pkg.defs	2003-08-05 20:13:34.000000000 +0200
@@ -61,7 +61,6 @@
   (remove-link GTK_HTML_COMMAND_REMOVE_LINK)
   (delete GTK_HTML_COMMAND_DELETE)
   (delete-back GTK_HTML_COMMAND_DELETE_BACK)
-  (set-mark GTK_HTML_COMMAND_SET_MARK)
   (disable-selection GTK_HTML_COMMAND_DISABLE_SELECTION)
   (bold-on GTK_HTML_COMMAND_BOLD_ON)
   (bold-off GTK_HTML_COMMAND_BOLD_OFF)
diff -Nur Gtk-Perl-0.7008/GtkHTML/xs/GtkHTML.xs Gtk-Perl-0.7008.test/GtkHTML/xs/GtkHTML.xs
--- Gtk-Perl-0.7008/GtkHTML/xs/GtkHTML.xs	2001-06-17 18:35:46.000000000 +0200
+++ Gtk-Perl-0.7008.test/GtkHTML/xs/GtkHTML.xs	2003-08-05 20:09:49.000000000 +0200
@@ -122,9 +122,10 @@
 	RETVAL
 
 void
-gtk_html_set_iframe_parent (html, parent)
+gtk_html_set_iframe_parent (html, parent, dummy)
 	Gtk::HTML	html
 	Gtk::Widget	parent
+        void*           dummy
 
 gulong
 gtk_html_begin (html)
@@ -167,8 +168,9 @@
 	bool	allow
 
 int
-gtk_html_request_paste (html, type, time)
+gtk_html_request_paste (html, dummy, type, time)
 	Gtk::HTML	html
+        void*           dummy
 	int	type
 	int	time
 
